translation_layer/jpl-capi.js

```diff
@@ -0,0 +1,150 @@
+/*globals console, window*/
+define ('main',['require','jquery','ExtendedModel','api/snapshot/adapters/BackboneAdapter','sim-common/Detect','sim-common/VersionCheck','text!sim-common/templates/notSupported.html','api/snapshot/adapters/BackboneAdapter','api/snapshot/Controller','api/snapshot/Transporter','api/snapshot/SimCapiValue'],function(require){
+
+    var $ = require('jquery');
+    var $body;
+
+    var ExtendedModel = require('ExtendedModel');
+    var adapter = require('api/snapshot/adapters/BackboneAdapter').getInstance();
+
+    var Capi = ExtendedModel.extend({});
+    var capi;
+
+    require('sim-common/Detect');
+    require('sim-common/VersionCheck');
+
+    var notSupportedTemplate = require('text!sim-common/templates/notSupported.html');
+    var BackboneAdapter = require ('api/snapshot/adapters/BackboneAdapter').getInstance();
+    var Controller = require('api/snapshot/Controller');
+    var Transporter = require('api/snapshot/Transporter').getInstance();
+    var Types = require('api/snapshot/SimCapiValue').TYPES;
+
+	var initialized = false;
+
+    window.receiveExposeFromRanger = function (name, type, value, allowedValues)
```

Comment:
sure, but you don't need to put functions in the global scope (on window), especially since you won't have access to the ranger code (so you won't be calling your functions outside of this code)